### PR TITLE
Wait for the right deployment name

### PR DESCRIPTION
**Pull request by davidfestal**
Signed-off-by: David Festal <dfestal@redhat.com>

### What does this PR do?

In the case of CRW, we should wait for the `codeready` deployment, instead of the `che` deployment by default.
It also chnage the expected name of other installation components such as the operator deployment name, service account, role bindings, etc ...

### What  issues does this PR fix or reference?

It should fix the issue https://issues.jboss.org/browse/CRW-462

### CODE REVIEW

File: src/common-flags.ts [hunk 1]
```diff
@@ -11,14 +11,14 @@ import { string } from '@oclif/parser/lib/flags'
 
 export const cheNamespace = string({
   char: 'n',
-  description: 'Kubernetes namespace where Che server is supposed by be deployed',
+  description: 'Kubernetes namespace where CodeReady Workspaces server is supposed by be deployed',
```
Comment (ibuziuk) [inline src/common-flags.ts]: `is supposed by be deployed` maybe you can also change it to ` is supposed **to** be deployed`

Comment (l0rd) [reply src/common-flags.ts]: He he good catch @ibuziuk but we should do the fix upstream first :-)

Comment (benoitf) [reply src/common-flags.ts]: https://github.com/che-incubator/chectl/pull/389

Comment (ibuziuk) [reply src/common-flags.ts]: Should we than backport it to 7.3.x ?

Comment (benoitf) [reply src/common-flags.ts]: https://github.com/che-incubator/chectl/pull/390

**Pull request merged**